src/main/java/com/amazonaws/services/dynamodbv2/datamodeling/AttributeEncryptor.java

```diff
@@ -32,13 +33,18 @@
 import com.amazonaws.services.dynamodbv2.datamodeling.encryption.TableAadOverride;
 import com.amazonaws.services.dynamodbv2.datamodeling.encryption.providers.EncryptionMaterialsProvider;
 import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
 
 /**
  * Encrypts all non-key fields prior to storing them in DynamoDB.
+ * <em>It is critically important that this is only used with @{link SaveBehavior#CLOBBER}. Use of
```

Comment:
Brevity for grokability?:

```
<em>This must be used with @{link SaveBehavior#CLOBBER}.</em> Use of any other...
```

Also, I think June would suggest using "can" rather than "may" for clarity, since it is a possibility, not a permission.